configure.in

```
@@ -2487,6 +2487,18 @@ if test "$wxUSE_ZLIB" != "no" ; then
             wxUSE_ZLIB=sys
         fi
     fi
+
+    dnl We should test if built-in zlib library is avalable, preferably
+    dnl downloaded through --recurse-submodules option if cloned from git.
+    dnl Would be available when downloaded as a zipped or tarball release.
+    if test "$wxUSE_ZLIB" = "builtin" ; then
+        AC_CHECK_FILE(
+            $ac_confdir/src/zlib/zlib.h,,
+            [AC_MSG_ERROR([
+        === Configured to use built-in ZLIB library, but couldn't find required
+        header. You probably need to run: git submodule update --init])
```

Would it be worth being even more precise with:
```suggestion
        header. You probably need to run: git submodule update --init src/zlib])
```
?

---

Sure